css/OWNERS

```diff
@@ -6,6 +6,7 @@
     {
       owners: [
         {name: 'ampproject/wg-components'},
+        {name: 'ampproject/wg-bento'},
         {name: 'ampproject/wg-performance'},
         {name: 'aghassemi', requestReviews: false},
```

Comment:
While you're in here, should `aghassemi` be removed? He left the project, no?

Reply:
Good question, will leave this for a different PR though.